.github/workflows/linux-build-and-test.yaml

```diff
@@ -40,9 +40,13 @@ jobs:
         sudo apt-get update
         sudo apt-get install flex bison lcov systemd-coredump gdb libipc-run-perl libtest-most-perl ${{ matrix.extra_packages }}
 
-    - name: Install macOS Dependencies using Perl
+    - name: Install macOS Dependencies
       if: runner.os == 'macOS'
       run: |
+        # This is needed because GitHub image macos-10.15 version
+        # 20210927.1 used OpenSSL 3.0 by mistake and we need 1.1.1, so
+        # we install openssl (which installs 1.1.1).
+        brew install openssl
```

Comment:
According to [this comment](https://github.com/actions/virtual-environments/issues/4205#issuecomment-932269485) it will install OpenSSL 3.0.

Recently we added support for OpenSSL 3.0: https://github.com/postgres/postgres/commit/72bbff4cd6eaf55239ccef79cec61766b5f8f1d2

Reply:
Description was wrong, fixed that.